templates/proxy.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment="HTTP_PROXY={{ http_proxy }}" "HTTPS_PROXY={{ https_proxy }}" "NO_PROXY={{ no_proxy }}"
```

Comment:
I'm confused about what this accomplishes without a `Type` or any of the `Exec*` fields?

Reply:
Basically, when adding a file to the service directory (which is different to the service unit file) https://github.com/charmed-kubernetes/charm-containerd/pull/2/files#diff-48a885c5068a45a19f138133911cd78cR209, it is inherited by the service unit file.

Reply:
Oh, I see.  There's an existing service and this is essentially patching it.  +1 very nice

Reply:
Yeah, it means we don't have to store a templated copy of the systemd unit, which means we aren't responsible for keeping that up to date :)